src/main/java/io/cdap/plugin/servicenow/source/util/ServiceNowConstants.java

```diff
@@ -155,5 +170,15 @@ public interface ServiceNowConstants {
    * The maximum number of retry attempts.
    */
   int MAX_NUMBER_OF_RETRY_ATTEMPTS = 5;
+
+  /**
+   * The INSERT operation
+   */
+  String INSERT_OPERATION = "insert";
+
+  /**
+   * The UPDATE operation
+   */
```

Comment:
this file should be moved under plugin/servicenow as it's now used by both the source and sink

Reply:
Moved to `plugin/servicenow` package 5d38574b7fa5eb4560278961cc05d061f5c47efb